src/Installers/Rpm/Runtime/Rpm.Runtime.rpmproj

```diff
@@ -14,7 +14,9 @@
   <PropertyGroup>
     <TargetFileName>$(RuntimeInstallerBaseName)-$(SharedFxVersion)-x64.rpm</TargetFileName>
     <CblMarinerTargetFileName>$(RuntimeInstallerBaseName)-$(SharedFxVersion)-cm.1-x64.rpm</CblMarinerTargetFileName>
+    <CblMariner2TargetFileName>$(RuntimeInstallerBaseName)-$(SharedFxVersion)-cm.2-x64.rpm</CblMariner2TargetFileName>
```

Comment:
nit: Might be worth setting `$(CblMarinerTargetFileName)`, `$(CblMariner2TargetFileName)`, `$(CblMarinerTargetPath)`, and `$(CblMariner2TargetPath)` in the Directory.Build.props file. Could go further and elminate most of this `<PropertyGroup/>`, just set a `$(RpmBaseName)` property in the project files.

Reply:
The `arch` is involved there for SharedFx but not Targeting Pack - I'll do that in the follow-up Linux Arm64 .rpm PR since that'll be dealing with arch anyways